README.md

```diff
@@ -11,7 +11,7 @@
 * [travis day two] (https://travis-ci.com/abbeymasters-401-advanced-javascript/lab-01/builds/128231592)
 
 #### Docs
-* [Docs] (docs)
+* [Docs] (https://abbeymasters-401-advanced-javascript.github.io/lab-01/docs)
 
 #### Running the app
 Commands:
```

Comment:
I'm not able to comment on your Errors.js file because it's not part of the changes in this pull request, but I noticed that your errors are basically duplicates of each other. The Model Error should be an error that is thrown when the data doesn't work for any reason (ie either casting OR missing fields, etc).